release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2995,6 +2995,63 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.8
+[id="ocp-4-18-8_{context}"]
+=== RHSA-2025:3577 - {product-title} {product-version}.8 bug fix update and security update
+
+Issued: 08 April 2025
+
+{product-title} release {product-version}.8 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:3577[RHSA-2025:3577] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:3579[RHBA-2025:3579] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.8 --pullspecs
+----
+
+[id="ocp-4-18-8-known-issues_{context}"]
+==== Known issues
+
+* IPsec is not supported on {op-system-base-full} compute nodes because of a `libreswan` incompatiblility issue between a host and an `ovn-ipsec` container that exist in each compute node. See (link:https://issues.redhat.com/browse/OCPBUGS-52949[*OCPBUGS-52949*]).
```

Comment:
```suggestion
* IPsec is not supported on {op-system-base-full} compute nodes because of a `libreswan` incompatiblility issue between a host and an `ovn-ipsec` container that exist in each compute node. (link:https://issues.redhat.com/browse/OCPBUGS-52949[*OCPBUGS-52949*]).
```

Reply:
:heavy_check_mark: